lib/release_notes_bot_web/views/captains_view.ex

```diff
@@ -47,8 +47,8 @@ defmodule ReleaseNotesBotWeb.CaptainsView do
       blocks: [
         Input.static_select(
           "client-select",
-          "static_select-action",
-          "Select Client",
+          "static_select-action:#{channel_id}",
```

Comment:
It'd be nice to know what parameters you're updating 😉

Reply:
I know this is quite hacky, but I needed to pass channel id into the modal. I figured that if the key starts with static text, I can just split the past that is static from the part that is not when parsing it after it is returned. A better solution would be to add an uneditable text field that has the id of the channel and the text with the channel name. This feeds into your comment that is right under this.

Reply:
I like that idea

Reply:
Okay so, slack modals do not appear to support disabled text fields. I will be revisiting this modal in the near future when more options are available to be configured. Is this fine for a first pass for now?

Reply:
So is this related to https://github.com/mojotech/captains-log-release-agent/pull/22/files#diff-56f72195c750dabdb4974eb2c888d0d09f06d937315dfeaddfb06b7d52bbffa3R56?

Reply:
Yes, directly related. This was the easiest way for me to get the slack channel ID over to the place where we need it (your link).